apps/table.html

```diff
@@ -446,6 +468,25 @@ <h3 class="text-center h3 mb-0">Available Slides</h3>
 		}
 	}
 
+	$(document).ready(function(){
+		$('#deleteModal').on('hidden.bs.modal', function (e) {
+			initialize();
+		  });
+	});
+	function deleteSld(e) {
+		const oid = e.dataset.id;
+		const store = new Store('../data/');
+		if (oid) {
+			$('#deleteModal').modal('toggle');
+			$("#confirmDelete").unbind( "click" );
+			$("#confirmDelete").click(function(){
+				store.deleteSlide(oid);
```

Comment:
Akhil I have implemented the same but here it returns a promise so, you can check if data.result.ok is 1 then initialize the table.